synfig-studio/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ endif()
 add_subdirectory(plugins)
 add_subdirectory(po)
 add_subdirectory(sounds)
-add_subdirectory(ui)
+add_subdirectory(resources)
 
```

Comment:
The goal is to move
- **synfig-studio/ui** folder to
- this folder **synfig-studio/src/gui/resources**

![Screenshot from 2020-11-24 17-29-37](https://user-images.githubusercontent.com/24214509/100091753-0b472880-2e7b-11eb-8010-0329bf980795.png)

Reply:
@Keyikedalube Ohh Sorry My Bad !! :(  I will fix this asap.

Reply:
BTW, @DhairyaBahl please build and test it on your computer from now on OK.

Seems Travis CI is gone. No longer free.

Reply:
@Keyikedalube I always test it on appveyor before pushing it as PR :)

Reply:
> Seems Travis CI is gone. No longer free.

I'm working on it. Good news soon)

Reply:
> I always test it on appveyor before pushing it as PR :)

Oh :)

Reply:
Resolved.